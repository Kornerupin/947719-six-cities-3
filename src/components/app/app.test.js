import App from './app';
import offersMock from '../../mocks/offers-mock';

it(`<App /> should render cards`, () => {
  const tree = renderer.create(<App offers={offersMock} onOfferNameClick={() => { }} />).toJSON();

  expect(tree).toMatchSnapshot();
});
