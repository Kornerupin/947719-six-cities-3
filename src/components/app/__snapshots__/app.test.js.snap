// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render <Main /> 1`] = `null`;
